www/src/main/java/SiteBuilder.java

```diff
@@ -0,0 +1,266 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we move this inside a package? `org.wso2.integration.ballerina...`

Reply:
Done